src/components/Settings/Notifications/NotificationsTelegram.tsx

```diff
@@ -0,0 +1,231 @@
+import React from 'react';
+import { Field, Form, Formik } from 'formik';
+import useSWR from 'swr';
+import LoadingSpinner from '../../Common/LoadingSpinner';
+import Button from '../../Common/Button';
+import { defineMessages, useIntl } from 'react-intl';
+import axios from 'axios';
+import * as Yup from 'yup';
+import { useToasts } from 'react-toast-notifications';
+import Alert from '../../Common/Alert';
+
+const messages = defineMessages({
+  save: 'Save Changes',
+  saving: 'Saving...',
+  agentenabled: 'Agent Enabled',
+  botAPI: 'Bot API',
+  chatId: 'Chat Id',
+  validationBotAPIRequired: 'You must provide a Bot API key.',
+  validationChatIdRequired: 'You must provide a Chat id.',
+  telegramsettingssaved: 'Telegram notification settings saved!',
+  telegramsettingsfailed: 'Telegram notification settings failed to save.',
+  testsent: 'Test notification sent!',
+  test: 'Test',
+  settinguptelegram: 'Setting up Telegram Notifications',
+  settinguptelegramDescription:
+    'To setup Telegram you need to <CreateBotLink>create a bot</CreateBotLink> and get the bot API key.\
+    Additionally, you need the chat id for the chat you want the bot to send notifications to.\
+    Which you can do by adding <GetIdBotLink>@get_id_bot</GetIdBotLink> to the chat or group chat.',
```

Comment:
```suggestion
    You can do this by adding <GetIdBotLink>@get_id_bot</GetIdBotLink> to the chat or group chat.',
```
Better flow written like this.